truss/templates/control/control/helpers/inference_server_process_controller.py

```diff
@@ -31,6 +30,8 @@ def __init__(
 
     def start(self):
         with current_directory(self._inference_server_home):
+            if os.path.isfile("inference_server_crashed.txt"):
```

Comment:
This may be better done in the inference server itself. It's best one entity controls the lifecycle of a file.

Reply:
Also, would be good to use an absolute path, this assumes that inference server and control server are started from the same path, which is true, but best not to couple those. We could use `~/.truss_inference_server_crashed.txt`. Another benefit is that that way we don't assume that cwd is writable, whereas we need to make home directory writeable for many other reasons (many python libs assume that).